components/Feature/Featured.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { ArrowUpward } from '@material-ui/icons'
+import styled from 'styled-components'
+import { featuredData } from '../dummyData'
+
+const FeaturedContainer = styled.div`
```

Comment:
Hey Dhanush, Please convert the CSS to a normal `.css` file for now only one file uses styled components I think we don't need it I will discuss and remove styled components.

Reply:
Yup sure,
Will change it to normal CSS.